src/main/java/com/java/guestdelluna/controller/ReviewController.java

```diff
@@ -35,7 +35,7 @@ public ModelAndView reviewDo(HttpServletRequest request , HttpServletResponse re
 	@RequestMapping(value="guestdelluna/reviewOk.do" , method=RequestMethod.POST)
```

Comment:
```suggestion
	@PostMapping("guestdelluna/reviewOk.do")
```
RequestMapping 으로 안하고 PostMapping 으로 해도 동작할텐데
`pom.xml` 보니까
- `java.version` : 1.6
- `org.springframework-version` : 3.1.1

이거 너무 낮아서 돌아갈지 모르겠음